language/tools/move-cli/tests/cli_testsuite.rs

```diff
@@ -3,13 +3,22 @@
 
 use move_cli::test;
 
-use std::path::Path;
+use std::path::PathBuf;
 
-pub const CLI_BINARY: &str = "../../../target/debug/move-cli";
-pub const CLI_TESTSUITE_DIR: &str = "tests/testsuite";
+pub const CLI_BINARY_PATH: [&str; 6] = ["..", "..", "..", "target", "debug", "move-cli"];
+pub const CLI_METATEST_PATH: [&str; 3] = ["tests", "metatests", "args.txt"];
 
-fn run_all(args_path: &Path) -> datatest_stable::Result<()> {
-    Ok(test::run_one(args_path, CLI_BINARY)?)
+fn get_cli_binary_path() -> String {
+    let pb: PathBuf = CLI_BINARY_PATH.iter().collect();
+    pb.to_str().unwrap().to_owned()
 }
 
-datatest_stable::harness!(run_all, CLI_TESTSUITE_DIR, r"args.txt$");
+fn get_metatest_path() -> String {
+    let pb: PathBuf = CLI_METATEST_PATH.iter().collect();
+    pb.to_str().unwrap().to_owned()
+}
+
+#[test]
+fn run_test() {
```

Comment:
This looks perfect. But don't we need another `#test` for cargo to run `tests/testsuite`? You can try changing a `.exp` file under `tests/testsuite` and running `cargo xtest` to see if it's running those tests.

Reply:
Ah, I think I might had a misunderstanding from the above discussion that the whole `test/testsuite` should not be involved in the CI process. In this case, I can add another `#test` for the `testsuite`, should be easy.